src/repositories/implementations/Prisma/LoginAdminPrismaRepository.ts

```diff
@@ -0,0 +1,29 @@
+import { PrismaClient } from '@prisma/client';
+import { AdminDTO, ILoginAdminRepository } from '../../interfaces';
+import { PrismaPostgresClient } from '../../../helpers/database/PrismaPostgresClient';
+
+export class LoginAdminPrismaRepository
+    extends PrismaPostgresClient
+    implements ILoginAdminRepository
+{
+    private prisma: PrismaClient;
+
+    constructor() {
+        super();
+        this.prisma = super.getPrismaClient();
+    }
+
+    async getAdminInfoByEmail(email: string): Promise<AdminDTO> {
```

Comment:
We should add some way to return if the admin exists, maybe null?
So instead of `Promise<AdminDTO>` it would be `Promise<AdminDTO | null>`
would also need a change in the UseCase to check that, I can do that if you want.